### PR TITLE
Bumps in ionic: gz-plugin3

### PR DESCRIPTION
**Pull request by azeey**
See https://github.com/gazebo-tooling/release-tools/issues/1027

### CODE REVIEW

**Review by scpeters — COMMENTED** [review]
I think we need to update the logic for setting `GZ_TOOLS_VER` at [CMakeLists.txt:48](https://github.com/gazebosim/gz-plugin/blob/669739582c097c5dcd0199231f4dca1e665039ca/CMakeLists.txt#L48)

**Comment by azeey** [comment]
> I think we need to update the logic for setting `GZ_TOOLS_VER` at [CMakeLists.txt:48](https://github.com/gazebosim/gz-plugin/blob/669739582c097c5dcd0199231f4dca1e665039ca/CMakeLists.txt#L48)

Merged #119 that took care of this.

File: test/integration/plugin.cc
Comment (iche033) [inline test/integration/plugin.cc:1]: I think the changes in this file are not needed

**Review by iche033 — APPROVED** [review]
one minor comment about unintended changes in test but they should be harmless

File: test/integration/plugin.cc
Comment (azeey) [reply test/integration/plugin.cc:1]: Thanks, reverted it in https://github.com/gazebosim/gz-plugin/pull/127/commits/534d846067a4f6b40db62c8585d2e8d4d28f9a7b

**Pull request merged**